client/js/lounge.js

```diff
@@ -276,9 +278,27 @@ $(function() {
 		return msg;
 	}
 
-	function buildChannelMessages(channel, messages) {
+	function appendMessage(container, chan, type, msg) {
+		if (!$(msg).hasClass("message") && type !== "lobby") {
```

Comment:
Hmm yeah, maybe multiple mode changes and kicks be a good idea to condense too. But whitelist is still safer in this case. So we don't unknowingly break anything if we add new features.